setup.cfg

```diff
@@ -49,6 +49,13 @@ docs =
     sphinx~=3.0
     sphinx-argparse-cli>=1.5
     sphinx-autodoc-typehints>=1.10
+lockminversions =
```

Comment:
Can we make this a constraint file rather than an extra group, please?

Reply:
Did you see my reasoning for the group? If you make it a constraint file, you can't ensure that is doesn't go out of sync, since it is not installed in the same command as the package. While if it's here, I'm pretty sure it does ensure it is in sync with the install_requires.

For example, let's say we start requiring `packaging>=20.9`, and we forget to update this. If it's a constrain file, either it will be installed first, and then the package will update packaging to 20.9 silently, or it's installed afterwords, and then it will be rolled back (and hopefully fail the tests, if we are lucky). While if it's an extra, it should immediately error out as unsolvable, with a clear description of why (`packaging>=20.9 ` and `packaging==19.0 ` in conflict). That was my thinking, anyway.

Reply:
> For example, let's say we start requiring `packaging>=20.9`, and we forget to update this.

If we set up things correctly (which we can) the installation will fail. @uranusjr you can invoke the constraint file with a package and ensure the above case does not happen, no?

Reply:
If we can make that happen, I'm happier with a constraint file, since this is not to be used externally (though, currently, there's no way to list a project's extras).

Reply:
Yes, I have something "working" (as in, if I manually break it by forcing too-old toml, it breaks :) ). Will commit momentarily.